File: src/main/java/cz/solutions/cockroach/EsppRecord.java
package cz.solutions.cockroach;

import lombok.Value;
import org.joda.time.LocalDate;

@Value
public class EsppRecord {
    LocalDate date;
    int quantity;
    double purchasePrice;
    double subscriptionFmv;
    double purchaseFmv;
    LocalDate purchaseDate;
}

File: src/main/kotlin/cz/solutions/cockroach/EsppRecord.kt
package cz.solutions.cockroach

import org.joda.time.LocalDate

data class EsppRecord(
    val date: LocalDate,
    val quantity: Double,
    val purchasePrice: Double,
    val subscriptionFmv: Double,
    val purchaseFmv: Double,
    val purchaseDate: LocalDate,
    val symbol: String = "",
    val broker: String = "",
)